platforms/minigame/common/engine/AssetManager.js

```diff
@@ -209,15 +209,32 @@ function downloadBundle (nameOrUrl, options, onComplete) {
     const version = options.version || cc.assetManager.downloader.bundleVers[bundleName];
     const suffix = version ? `${version}.` : '';
 
+    function getConfigPathForSubPackage () {
+        if (sys.platform === sys.Platform.TAOBAO_MINI_GAME) {
+            return `${bundleName}/config.${suffix}json`;
+        }
+        return `subpackages/${bundleName}/config.${suffix}json`;
```

Comment:
Why `taobao minigame` doesn't need the `subpackages` prefix?

Reply:
taobao mini game depends on config of game.json, subpackages has been config.
And taobao's filesystem does not support other directory before bundle name

Reply:
loadsubpackage's example for taobao.
![image](https://user-images.githubusercontent.com/3078423/221111088-34f158d4-ceb8-4110-a8cf-f7764e9fed72.png)